src/CoreWCF.Http/tests/CoreWCF.Http.Tests.csproj

```diff
@@ -49,4 +49,10 @@
     <ProjectReference Include="..\src\CoreWCF.Http.csproj" />
     <ProjectReference Include="..\..\CoreWCF.BuildTools\src\CoreWCF.BuildTools.Roslyn4.0.csproj" OutputItemType="Analyzer" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' != 'net8.0'">
```

Comment:
When doing this, I like to include the files with a \<None\> item so they still show up in VS solution explorer.